scripts/snapshot-controller-psp.yaml

```diff
@@ -19,9 +19,8 @@ spec:
   volumes:
     - "configMap"
     - "emptyDir"
-    - "projected"
     - "secret"
-    - "downwardAPI"
+    - "projected"
```

Comment:
@Madhu-1 is it failing without `projected` ?

Reply:
Tested with multiple kubernetes versions, Yes it's failing without projected and secrets

Reply:
hmmm :(, I dont see a reason to have a `projected` volumes here, however if its failing without that , lets live with it .. thanks for the clarification.